test/unit/core.js

```diff
@@ -38,19 +38,21 @@ QUnit.test( "jQuery()", function( assert ) {
 	// few here but beware of modular builds where these methods may be excluded.
 	if ( includesModule( "deprecated" ) ) {
 		expected++;
-		attrObj[ "click" ] = function() { assert.ok( exec, "Click executed." ); };
+		attrObj.click = function() {
+ assert.ok( exec, "Click executed." );
+};
```

Comment:
Formatting here doesn't look right. A similar mistake is repeated in many, many places, please fix them all.

Reply:
I thought I got them all, but I think I only fixed them on my 3.x branch.